eventuals/task.h

```diff
@@ -385,23 +400,46 @@ struct _TaskFromToWith final {
 
       value_or_dispatch_ = [f = std::move(f)](
                                Action action,
-                               std::optional<std::exception_ptr>&& exception,
+                               std::optional<
+                                   typename VariantOfStoppedAndErrors<
+                                       Catches_>::type>&& error,
                                Args_&... args,
                                std::optional<MonostateIfVoidOr<From_>>&& arg,
-                               std::unique_ptr<void, Callback<void(void*)>>& e_,
+                               std::unique_ptr<
+                                   void,
+                                   Callback<void(void*)>>& e_,
```

Comment:
This formatting change looks unnecessary, can we please revert?

Reply:
exceed 80 characters

Reply:
It doesn't look like it to me!